scipy/optimize/_linprog.py

```diff
@@ -168,7 +168,7 @@ def linprog_terse_callback(res):
 
 def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
             bounds=None, method='interior-point', callback=None,
-            options=None, x0=None):
+            options=None, x0=None, integrality=None):
     r"""
     Linear programming: minimize a linear objective function subject to linear
```

Comment:
Should the short docstring mention anything about integral constraints?  (i.e. non-linear constraints)  I suppose if all solvers don't support this, then better to only include in highs methods?

Reply:
Let's not add that to the short docstring. We'll write it in `milp`, of course, and this will help users distinguish between the two.